common/app/views/support/ImageProfile.scala

```diff
@@ -12,6 +12,7 @@ import org.apache.commons.math3.fraction.Fraction
 import org.apache.commons.math3.util.Precision
 import common.Environment.{app, awsRegion, stage}
 import play.api.libs.json.{Json, Writes}
+import views.support.EmailImage.width
```

Comment:
this probably isn't needed  intellij accidental import! but can be cleaned up in future